ctl/src/command.rs

```
@@ -591,10 +586,9 @@ pub fn metrics(mut channel: Channel<CommandRequest,CommandResponse>, json: bool)
           },
           CommandStatus::Error => {
             if json {
-              print_json_response(&message.message);
+              print_json_response(&message.message)?;
```

Same here, you should return an error

---

How about we do

```rust
print_json_response(&message.message)
```
with no `?` or `;` ?

---

nevermind, I corrected as above